docs/pages/router/advanced/singular.mdx

```diff
@@ -82,14 +82,16 @@ The `navigate` and `push` events apply the singular constraint differently.
 
 <FileTree files={['app/_layout.tsx', 'app/index.tsx', 'app/[profile].tsx']} />
 
-Using the app structure above, if we have the history of
+Using the app structure above, if you have the history of:
 
 - `/profile-1`
 - `/profile-2`
 - `/profile-2`
 - `/profile-3`
 - `/profile-2`
 
+If you navigate to `/profile-2`, the singular constraint will not be applied as the current route is the target route when using `navigate`. However, if you push `/profile-2`, the singular constraint will be applied.
```

Comment:
@marklawlor, it felt that there was some explanation missing just before the following example. Hoping it's the correct explanation.